marked-element.html

```diff
@@ -253,7 +253,8 @@
     /**
      * Unindents the markdown source that will be rendered.
      *
-     * @param {string} text
+     * @param {!string} text
```

Comment:
Don't actually need it for primitives like `string`, they are already non-null by default. :)

https://github.com/google/closure-compiler/wiki/Annotating-JavaScript-for-the-Closure-Compiler#nullable-type

Reply:
TIL :+1: